Pokemon/Pokemon/Models/Pokemon.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Pokemon.swift
+//  Pokemon
+//
+//  Created by Cora Jacobson on 7/18/20.
+//  Copyright © 2020 Cora Jacobson. All rights reserved.
+//
+
+import Foundation
+
+struct Pokemon: Codable {
+    
+    enum Keys: String, CodingKey {
+        case name
+        case id
+        case types
+        case abilities
+        case imageURL = "sprites"
+        
```

Comment:
Add an imageString key to account for that property since it is named differently than imageURL

``` swift
case imageString
```